test/example/widget_test.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:dartz/dartz.dart';
```

Comment:
This is the test, it could have been broken down into multiple test but if this was a production app I find that it is helpful to have a test that covers a users journey through the app.